.github/workflows/apidiff.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v2
```

Comment:
Thank you for the contribution, but see that it still missing be updated in some place : https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kubebuilder/2561/pull-kubebuilder-test/1507421392423882752

Reply:
See: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml#L13

Reply:
I haven't ;-)
See this PR
https://github.com/kubernetes/test-infra/pull/25775